packages/z3/z3.4.8.8-1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "simon@aestheticintegration.com"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+patches: ["4468.patch"]
+build: [
+  [ "python2.7" "scripts/mk_make.py" "--ml" "--staticlib" ]
+  [ make "-C" "build" "-j" jobs ]
+]
+install: [
+  [ "sh" "-c" "ocamlfind install z3 build/api/ml/* build/libz3-static.a"]
+  [ "cp" "build/z3" "%{bin}%/z3"]
+]
+
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+  "zarith"
+  "conf-gmp"
+  "conf-python-2-7" {build}
+]
+synopsis: "Z3 solver"
+extra-files: [
+  ["4468.patch" "md5=05a00fdd98aecba2198393bff8484efa"]
+]
+
+remove: [
+  ["ocamlfind" "remove" "z3"]
+  ["rm" "-f" "%{bin}%/z3"]
+]
+
```

Comment:
In your first comment you said you introduced the remove field. However this field should not be necessary since opam 2.0 unless you specifically require to remove something that has been installed at runtime (after installation), and it does not seem to be the case here. I'm going to remove it and this PR should be good to go